designer/client/src/list/ListEdit.tsx

```diff
@@ -193,6 +193,44 @@ export function ListEdit() {
           }
           errorMessage={errors.title}
         />
+        <Radios
+          id="list-type"
+          name="type"
+          hint={{ children: i18n('list.typeHint') }}
+          fieldset={{
+            legend: {
+              className: 'govuk-fieldset__legend--s',
+              children: [i18n('list.type')]
+            }
+          }}
+          value={selectedList.type}
+          defaultValue="string"
+          items={[
+            {
+              children: ['String'],
+              value: 'string',
+              hint: {
+                children: [i18n('list.typeHintString')]
+              },
+              disabled: !selectedList.isNew
```

Comment:
Thanks for pushing this up Dave

Since screen readers can't focus + announce disabled form fields, can we change this to text?

E.g. Just a heading and a description instead:

> **List type**
> Text values only

> **List type**
> Numeric values only

Save the non-disabled form fields for when creating new lists only

Let me know if you need a hand running it through `validate()`